scripts/image_undistortion.py

```diff
@@ -36,8 +36,6 @@ def __init__(self):
         self.mf.registerCallback(self._img_callback)
         self._pub_rect_img_l = rospy.Publisher("/camera_l/image_rect_color", Image, queue_size=1)
         self._pub_rect_img_r = rospy.Publisher("/camera_r/image_rect_color", Image, queue_size=1)
```

Comment:
```suggestion
        self._pub_rect_img_r = rospy.Publisher("/camera_r/image_rect_color", Image, queue_size=1)
        rospy.wait_for_message('/csi_cam_0/image_raw', Image)
```
こういうときどうするのがいいのか未だよくわかってないのですが、rospy.wait_for_message入れるとエラーは出なくなりました。